support/support.go

```diff
@@ -520,3 +520,62 @@ func sanitizeEnv(kvs map[string]string) {
 		}
 	}
 }
+
+func Summarize(b *Bundle) (msgs []string) {
+	if b == nil {
+		return []string{}
+	}
+	if b.Network.Netcheck == nil {
+		msgs = append(msgs, "Netcheck missing from bundle!")
+	} else {
+		if b.Network.Netcheck.Error != nil {
+			msgs = append(msgs, "Client netcheck: "+*b.Network.Netcheck.Error)
+		}
+		for _, warn := range b.Network.Netcheck.Warnings {
+			msgs = append(msgs, "Client netcheck: "+warn.String())
+		}
+	}
+
+	if b.Deployment.HealthReport == nil {
+		msgs = append(msgs, "Deployment health check missing from bundle!")
+	} else {
+		if b.Deployment.HealthReport.AccessURL.Error != nil {
+			msgs = append(msgs, "Deployment health: "+*b.Deployment.HealthReport.AccessURL.Error)
```

Comment:
Wouldn't it be better to replace `Deployment health:` with a section name, for example `Access URL:`?

Otherwise, we will get a large block of 

`Deployment health: `
`Deployment health: `
`Deployment health: `
`Deployment health: `
`Deployment health: `

Same question for all sections.

Reply:
Going to take this out for now and address in a separate PR; there's some refactoring we may need to do for this.

Reply:
https://github.com/coder/coder/pull/12963